R/dv8.R

```diff
@@ -0,0 +1,153 @@
+#' Convert from sdsm.json to a dependency matrix in sdsm.dv8-dsm file
```

Comment:
The title of the function docs is not consistent to the function name (this is something I noted in your group prior review) and reinforced in e-mail and meetings. E.g.:

https://github.com/sailuh/kaiaulu/pull/171#discussion_r1151623728

The equivalent here would be to convert from a JSON dsm to a binary DSM. Moreover, note your function is not converting just a "sdsm" but both sdsm and hdsm, which is why I omitted the mention on the line above.

Moreover, the body of the function documentation is missing. In it, you should refer to the lingo DV8 calls these files instead.

<img width="770" alt="Screen Shot 2020-06-09 at 1 10 38 AM" src="https://user-images.githubusercontent.com/17270563/231047066-ed569edb-1d8f-4dd8-a3f4-0943447333bf.png">

Therefore, the body of your function documentation must state it converts both a hdsm.json and sdsm.json to a hdsm.dv8-dsm and a -sdsm.dv8-dsm 

This makes it so a user can understand what this function does from Kaiaulu nomenclature, but also from DV8 nomenclature.

Documentation surrounding DV8 is vital as I noted before, so please make sure all functions abide by this standard in your Pull Request.